akka-docs/src/main/paradox/typed/mailboxes.md

```diff
@@ -83,20 +83,7 @@ be used.
 
 ### Default Mailbox
 
-The default mailbox is used when the mailbox is not specified.
-This is an unbounded mailbox, backed by a
-`java.util.concurrent.ConcurrentLinkedQueue`.
-
-`SingleConsumerOnlyUnboundedMailbox` is an even more efficient mailbox, and
-it can be used as the default mailbox, but it cannot be used with a BalancingDispatcher.
-
-Configuration of `SingleConsumerOnlyUnboundedMailbox` as default mailbox:
-
-```
-akka.actor.default-mailbox {
-  mailbox-type = "akka.dispatch.SingleConsumerOnlyUnboundedMailbox"
-}
-```
```

Comment:
maybe we still need to mention this somewhere in the Classic Mailboxes (/mailboxes.md) docs

Reply:
It's there, I didn't remove it from that one: https://doc.akka.io/docs/akka/current/mailboxes.html#default-mailbox

Reply:
https://doc.akka.io/docs/akka/2.6/mailboxes.html#default-mailbox

We have done massive restructuring of docs, moving much content to the `/typed/*.md` and `/*.md` is for Classic.

Since the Default Mailbox is now different it makes sense to put this section back to /mailboxes.md

Reply:
Doh, gotcha